client/src/components/MultiJurisdictionAudit/Setup/Review/index.tsx

```diff
@@ -279,7 +288,9 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
                   locked ||
                   !isSetupComplete(jurisdictions, contests, auditSettings)
                 }
-                onClick={handleSubmit}
+                onClick={() => {
+                  handleSubmit()
+                }}
```

Comment:
unnecessary change?

Reply:
@jonahkagan Oh good catch. I was adding in a console.log there to check that it was submitting when I was testing things, but forgot to fix it all the way when I removed the log.